build.cmd

```diff
@@ -403,6 +403,7 @@ set "__BuildLog=%__LogsDir%\MScorlib_%__BuildOS%__%__BuildArch%__%__BuildType%.l
 set "__BuildWrn=%__LogsDir%\MScorlib_%__BuildOS%__%__BuildArch%__%__BuildType%.wrn"
 set "__BuildErr=%__LogsDir%\MScorlib_%__BuildOS%__%__BuildArch%__%__BuildType%.err"
 set __msbuildLogArgs=^
+/p:UseOpenSourceSign=false ^
```

Comment:
Actually I just checked and it looks like the msbuildargs doesn't understand the `=` sign on the property so I am going to have to make a specific conditional argument as @gkhanna79 suggested. I'll update this PR accordingly.